packages/wrangler/src/__tests__/version.test.ts

```diff
@@ -0,0 +1,26 @@
+import { mockConsoleMethods } from "./helpers/mock-console";
+import { useMockIsTTY } from "./helpers/mock-istty";
+import { runWrangler } from "./helpers/run-wrangler";
+
+describe("version", () => {
+	const std = mockConsoleMethods();
+	const { setIsTTY } = useMockIsTTY();
+
+	// We cannot test output of version banner,
+	// as it is disabled in jest environments
+
+	// it("should output version banner", async () => {
+	// 	await runWrangler("-v");
+	// 	expect(std.out).toMatchInlineSnapshot(`
+	// 	" ⛅️ wrangler 2.0.22
+	// 	[38;5;214m--------------------[39m"
+	// `);
+	// });
+
+	it("should output current version if !isTTY", async () => {
+		setIsTTY(false);
+
+		await runWrangler("-v");
+		expect(std.out).toMatchInlineSnapshot(`"2.0.22"`);
```

Comment:
This will actually fail when we do a release. You should replace the actual version number in std.out with something like XXX and populate the snapshot

Reply:
You can take from my bundle size workaround @cameron-robey